[2021-02-16 12:11:29,015] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-02-15T00:00:00+00:00 [queued]>
[2021-02-16 12:11:29,640] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-02-15T00:00:00+00:00 [queued]>
[2021-02-16 12:11:29,662] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-16 12:11:29,670] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-16 12:11:29,687] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-16 12:11:30,860] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_0> on 2021-02-15T00:00:00+00:00
[2021-02-16 12:11:31,060] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', '2021-02-15T00:00:00+00:00', '--job-id', '12', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpyaeddu5r', '--error-file', '/tmp/tmpue_v91oj']
[2021-02-16 12:11:31,074] {standard_task_runner.py:77} INFO - Job 12: Subtask runme_0
[2021-02-16 12:11:30,983] {standard_task_runner.py:52} INFO - Started process 59 to run task
[2021-02-16 12:11:32,405] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_0 2021-02-15T00:00:00+00:00 [running]> on host 071a2d8d0e8e
[2021-02-16 12:11:32,939] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_0
AIRFLOW_CTX_EXECUTION_DATE=2021-02-15T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-15T00:00:00+00:00
[2021-02-16 12:11:33,010] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-02-16 12:11:33,018] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_0__20210215" && sleep 1
[2021-02-16 12:11:33,069] {bash.py:169} INFO - Output:
[2021-02-16 12:11:33,072] {bash.py:173} INFO - example_bash_operator__runme_0__20210215
[2021-02-16 12:11:34,078] {bash.py:177} INFO - Command exited with return code 0
[2021-02-16 12:11:34,687] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20210215T000000, start_date=20210216T121129, end_date=20210216T121134
[2021-02-16 12:11:36,263] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-16 12:11:36,443] {local_task_job.py:146} INFO - Task exited with return code 0
